src/SpectatorView.Unity/Assets/SpectatorView/Scripts/Compositor/TextureManager.cs

```diff
@@ -19,6 +19,13 @@ public class TextureManager : MonoBehaviour
 #if UNITY_EDITOR
         public CompositionManager Compositor { get; set; }
 
+        /// <summary>
+        /// Gets or sets whether or not the quadrant video output is required.
+        /// The quadrant frame will not be rendered when this is not set and when
+        /// the video recording mode does not require it.
+        /// </summary>
+        public bool IsQuadrantVideoOutputRequired { get; set; }

```

Comment:
>IsQuadrantVideoOutputRequired [](start = 20, length = 29)

to remain consistent with the flag below, can we call this IsVideoOutputQuadrantMode?

Reply:
I changed the name to "IsQuadrantVideoFrameNecessary". The quadrant frame might be necessary because it's required for video recording, or because it's being previewed on screen. The output is never "quadrant" from the capture card

---
In reply to: [310673196](https://github.com/microsoft/MixedReality-SpectatorView/pull/172#discussion_r310673196) [](ancestors = 310673196)